svr-runopts.c

```diff
@@ -448,13 +448,14 @@ void svr_getopts(int argc, char ** argv) {
 	}
 	else
 	{
-		size_t i;
-		for (i = 0u; i < strlen(svr_opts.allowed_max_size); ++i)
+		size_t i = 0u;
+		while ((svr_opts.allowed_max_size[i] != '\0') && (isdigit(svr_opts.allowed_max_size[i]) != 0))
```

Comment:
consider isdigit() to return "a boolean". Don't compare its result.

Reply:
Done.